components/family-studies/ui/src/main/resources/PhenoTips/FamilyRecordField_memberReports.xml

```diff
@@ -116,7 +116,7 @@
 #end
 #foreach ($patient in $family.members)
   #if ($patient)
-    #set ($patientDoc = $xwiki.getDocument($patient.document))
```

Comment:
should we also change [here](https://github.com/phenotips/phenotips/blob/master/components/family-studies/ui/src/main/resources/PhenoTips/FamilyContentTopMenu.xml#L493) and [here](https://github.com/phenotips/phenotips/blob/master/resources/ui/src/main/resources/PhenoTips/PatientPhenotypeExportService.xml#L64) ?